### DISCUSSION TITLE
中文文档中有个位置内容有误

### DISCUSSION THREAD

**Discussion by Mrzhang06** [ppoffice/hexo-theme-icarus#1188] [General]
文档地址：[自定义Hexo标签插件](https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E8%87%AA%E5%AE%9A%E4%B9%89hexo%E6%A0%87%E7%AD%BE%E6%8F%92%E4%BB%B6/#tab_align_right_1)
如下图，图中蓝色位置，配置值应为 **right** 吧？
![image](https://user-images.githubusercontent.com/45694599/219454387-83128818-5328-4cae-958e-998b37220097.png)

**Comment by ppoffice** [comment]
Fixed in 0132965ca3204761a2b0e89a29ec4979aa914cb6.
